small_language_model_pretraining/nemo/config_H200_1x8x1_8b.sh

```diff
@@ -0,0 +1,111 @@
+export MAX_RETRIES=1
+
+# Folder mapping:
+# Output directory that holds logs, any path that you like. 
+export JOB_DIR="/workspace/code/logs"
+# Image / container path, either local cache file or remote URL
+export IMAGE="DUMMY"
+# Dataset: C4 dataset location that contains the dataset after preprocessing
+# export ORIGINAL_C4_PATH="/data/data/C4"
+
+# This corresponds to the PREPROCESSED_PATH in README section 3's dataset download part
+export PREPROCESSED_PATH="/data/llama3_8b/data/C4_processed"
+export MERGED_C4_PATH="/data/llama3_8b/data/C4_merged"
+# Dataset: Numpy index working directory, contains shuffled dataset
+# This path must be able to hold >400GB data
+export TMP_NPY_INDEX="/data/npy_indices"
+# Dataset: Tokenizer path
+# This corresponds to the TOKENIZER_PATH in README section 3's tokenizer download part
+export TOKENIZER_PATH="/data/llama3_8b/model/Llama-3.1-8B"
+# export TOKENIZER_PATH="/data/llama3_405b_ref/tokenizer"
+
+# Model: checkpoint and tokenizer path
+#     This is the checkpoint that we want to start with. 
+#     Each checkpoint should be a folder containing two sub-folders: context and weights. 
+#     And we need to pass this folder's path (the folder containing context and weights) here.  
+export MODEL_CKPT="/data/llama3_8b/model/Llama-3.1-8B_nemo"
+# export MODEL_CKPT="None"
+# Model: Continual checkpoint directory to write and resume
+#     This is the directory to hold all intermediate checkpoints. 
+#     Once a run is complete and we specify to save checkpoints, 
+#     we should see a checkpoint written in this folder
+#     with name `checkpoint-par-x-y-steps`
+#     Inside this directory, there should be a `checkpoint` directory that holds context and weights
+#     which is the "actual checkpoint". 
+#     Notice that this path must be able to hold at least 5.2TB data since each checkpoint is 5.2TB. 
+export CONTINUAL_CKPT="/data/model/saved_ckpts"
+# Model: Whether we want to restore from MODEL_CKPT path. If 0, then we are not restoring. 
+export USE_CKPT=0
+# Model: Whether we are resuming from a NeMo-formatted HuggingFace checkpoint (weights only). 
+#     If set to 1, then checkpoint resuming code will not try to load the optimizer states. 
+export FROM_HF=1
+# Model: Whether we want to save a checkpoint. Must be 1 if NPAR > 1. If 1, then we save a checkpoint at the end.
+export SAVE_CKPT=0
+
+# export SRC_PATH="/data/training/Llama-3.1-8B/original/"
+# export DST_PATH="/data/training/Llama-3.1-8B/nemo/"
+export CONT_IMAGE_URL="rocm/mlperf:llama31_8b_training_5.1_gfx942 "
+
+
+# Training Configs: 
+# Model: size, to choose from 8b, 70b, 405b
+export SIZE="8b"
+# Dataloader: Global batch size
+export GBS=32
+# Dataloader: Micro batch size
+export MBS=2
+# Dataloader: Max run N batches, optional
+#     If an empty string is provided (""), then the training will continue until time limit
+#     If we want to save a checkpoint, then this value must be set
+export MAX_STEPS=1200000
```

Comment:
Remove all 4 values and use defaults as set in pretrain_llama31.py

export MAX_STEPS=1200000
export WARMUP_STEPS=512 # 16384 // GBS
export EVAL_EVERY=384 # 12288 // GBS
export START_EVAL_AT=384 #